src/containers/InboxPage/InboxPage.module.css

```diff
@@ -297,6 +297,10 @@
   margin-bottom: 0;
 }
 
+.itemSeats {
+  margin-top: 3px;
```

Comment:
This should be part of css grid!

In itemLink:
```css
 grid-template-areas:
    'dot name    state'
    '... item    state'
    '... details state';
    '... seats   state';
```

Here, perhaps something like this:

```css
  composes: marketplaceTinyFontStyles from global;
  grid-area: seats;
  color: var(--colorGrey700);
  margin-top: 0;
  margin-bottom: 0;
```

Disclaimer: Untested code!